DigitalOffice/Services/UserService/Database/Entities/UserAchievement.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace UserService.Database.Entities
+{
+    public class UserAchievement
+    {
+        public Guid UserId { get; set; }
+        public virtual User User { get; set; }
+        public Guid AchievementId { get; set; }
+        public virtual Achievement Achievement { get; set; }
```

Comment:
Don't forget that such navigational properties should also be checked in Achievement entity and User entity.

Since this works as a link between users and achievements, please, look up into many-to-many relationship in EF Core.